plugins/flytekit-bigquery/tests/test_agent.py

```diff
@@ -90,7 +90,7 @@ def __init__(self):
     resource = agent.get(metadata)
     assert resource.phase == TaskExecution.SUCCEEDED
     assert (
-        resource.outputs.literals["results"].scalar.structured_dataset.uri
+        resource.outputs["results"].uri
```

Comment:
Technically, if there is another library that depends on `resource.outputs.literals` being defined, then this breaks backward compatibility.

I do not think there are anyone that depends on this, so I am okay with this change.

Reply:
Thank you so much for thinking so carefully